pages/loginPage.js

```diff
@@ -0,0 +1,7 @@
+export class LoginPage {
+    loginMenu = "//a[@id='login2']";
+    loginUserNameInput = "//input[@id='loginusername']";
+    loginPasswordInput = "//input[@id='loginpassword']";
```

Comment:
```suggestion
    loginUserNameInput = "//input[@id='loginusername']";
    loginPasswordInput = "//input[@id='loginpassword']";
```

you can write locators names shortly like  userNameField and passwordField